portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +34,35 @@ function createCommentElement(userName, comment) {
   divElement.innerText = userName + ': ';
   divElement.appendChild(pElement);
   return divElement;
-}
\ No newline at end of file
+}
+
+function getLoggedInStatus() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    const loggedIn = json.loggedInStatus;
+
+    if (loggedIn){
+      const pElement = document.getElementById('display-user');
+      pElement.style.display = "block";
+      pElement.textContent = "Logged in as: " + json.userEmail;
```

Comment:
Maybe rename to displayName from userEmail

Reply:
Will do when I refactor code as an update to PR #14